pkgs/development/tools/knightos/scas/default.nix

```diff
@@ -4,13 +4,13 @@
 stdenv.mkDerivation rec {
   pname = "scas";
 
-  version = "0.4.6";
```

Comment:
I wonder if it might be better to wait until 1.0.0 is out. In the meanwhile, there also seems to be 0.5.0.